src/main/java/com/notably/logic/parser/EditCommandParser.java

```diff
@@ -24,13 +24,14 @@
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_BODY);
 
+        String body;
         if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_BODY)
                 || !argMultimap.getPreamble().isEmpty()) {
-            throw new ParseException(String.format("Invalid Command"));
+            body = args.trim();
```

Comment:
Let's check for an empty title. That is, throw a `ParseException` if the user types `edit` only, without any arguments!

Reply:
I think this is ok, lets give the user to option to make the body empty.